### PR TITLE
Add schema and validation workflow

### PR DESCRIPTION
**Pull request by zkxs**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# https://EditorConfig.org
+root = true
+
+[*.yml]
+indent_style = space
+indent_size = 2
+end_of_line = lf
```
Comment (ljoonal) [inline .editorconfig:7]: Move the duplicate definitions under [*]?

Comment (zkxs) [reply .editorconfig]: done

**Comment by zkxs** [comment]
Okay, I've finished my workflow debugging nightmare and this should be good to review.

**Review by ljoonal — APPROVED** [review]

**Comment by zkxs** [comment]
Time to see if this runs for existing PRs...

**Pull request merged**